tests/test_db_utils.py

```
@@ -20,7 +20,8 @@
 import sys
 import tempfile
 import unittest
-from typing import Any
+from contextlib import contextmanager
+from typing import Any, Generator
```

Ruff (0.15.10)

[warning] Import from `collections.abc` instead: `Generator`

Import from `collections.abc`

(UP035)